alf/trainers/policy_trainer.py

```diff
@@ -101,8 +101,9 @@ def __init__(self, config: TrainerConfig):
         self._random_seed = config.random_seed
         self._num_iterations = config.num_iterations
         self._num_env_steps = config.num_env_steps
-        assert self._num_iterations + self._num_env_steps > 0, \
-            "Must provide #iterations or #env_steps for training!"
+        assert (self._num_iterations + self._num_env_steps > 0
+                and self._num_iterations * self._num_env_steps == 0), \
```

Comment:
These two need to be exclusive because otherwise the checkpoint interval will function incorrectly.